.github/workflows/test-pythonpath-ubuntu.yml

```diff
@@ -38,6 +38,11 @@ jobs:
         shell: bash
         run: |
 
+          # WARNING: Do NOT 'export PYTHONPATH=...'. Environment variables seem
```

Comment:
Apparently `export PYTHONPATH=...` alters environments for other GitHub action jobs – even those not defined within this file! Instead the variable is set only during the `run-tests.sh` call.